iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/LimitViolationUtils.java

```diff
@@ -12,9 +12,7 @@
 import com.powsybl.iidm.network.limitmodification.result.AbstractDistinctLimitsContainer;
 import com.powsybl.iidm.network.limitmodification.result.LimitsContainer;
 
-import java.util.Collection;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
Is this on purpose ?
If not maybe use these IntelliJ settings https://www.baeldung.com/intellij-disable-wildcard-import

It is usually cleaner to explicitely import classes in case of synonims in different packages.

Reply:
ok I'll change that